docs/performance/database-indexes.rst

```diff
@@ -0,0 +1,20 @@
+Database indexes
+================
+
+Indexing tables allows you to decrease fetching time from database.
+
+As an example lets take a look on customers list
+
+Default index page is sorted by registration date, to create table index all you need to do is modify `Customer` Entity and add index using annotations.
```

Comment:
```suggestion
The default index page is sorted by registration date, to create a table index all you need to do is modify `\App\Entity\Customer\Customer` entity and add the index using annotations. This class can be found at `src/Entity/Customer/Customer.php`. Once open, paste the following code:
```

Reply:
@lchrusciel why once the slashes are forward and once backward?

Reply:
one was the file path, while the other FQCN